.gitignore

```diff
@@ -9,3 +9,6 @@ node_modules
 
 .DS_store
 wrangler.toml
+
+## vscode
+launch.json
```

Comment:
unless you store any secrets in this file, I would be fine including it in git for new developers to have better VSCode integration

Reply:
Ok, sounds good. Added here: https://github.com/StampyAI/stampy-ui/pull/193/commits/0307b767e45c247aed533982e6d45fa958df9661